.travis.yml

```
@@ -3,4 +3,4 @@ group: stable
 cache: bundler
 language: ruby
 rvm:
-  - 2.3.3
+  - 2.6.3
```

I guess this was meant to be updated in the past however now that we are at 2.7.2 of rvm, this is out of date and therefore this file probably doesn't need modification.
```suggestion
  - 2.7.2
```